conda_vendor/conda_vendor.py

```diff
@@ -123,19 +116,31 @@ def _get_environment_name(environment_file: Path) -> str:
 
 
 def _get_virtual_packages(
```

Comment:
conda_vendor.py is pretty big. May help other contributors if you separate out some stuff, such as virtual package-related funcs.

Reply:
i want to refactor more completely in the future, to separate the library stuff from the ui stuff. 

the counter argument is that it's ~800 lines with comments, ~600 lines of code (with at least 200 lines just doing ui), all in all, not too big. 

i'm not sure how much easier it is to understand if we just hide stuff away, and all the code kind of depends on the other code there, i couldn't find good module separation, yet.  

right now, all of conda vendor sits in conda_vendor.py which means there are no secrets.

Reply:
I think refactoring can wait.